docs/reference/commands.md

```diff
@@ -400,7 +401,7 @@ List all components created in this project.
 - `--columns COLUMNS`
     - Comma-separated list of columns to show. Available columns: name, full_name, path. Defaults to `--columns full_name,path`.
 - `-s`, `--simple`
-    - Only show table data, without headers. Use this option for generating machine-readable output.
+    - Only show table data, without headers. Use this option for generating machine- readable output.
```

Comment:
This looks like an error.

Reply:
Yeah, these files in `docs/reference/` are auto-generated and are overwritten when generating docs anyway, so it's kinda just noise. But if these files are totally omitted from the source code, then these pages will be missing in the documentation site. So I just include the changes after running `mkdocs serve` locally :D